config/locales/petitions.en-GB.yml

```diff
@@ -68,8 +68,8 @@ en-GB:
           other: "<span class=\"count\">%{formatted_count}</span> signatures in the last hour"
       in_your_constituency:
         html:
-          one: "%{formatted_count} signature from your constituency"
-          other: "%{formatted_count} signatures from your constituency"
+          one: "%{formatted_count} signature from "
+          other: "%{formatted_count} signatures from "
```

Comment:
We can specify %{constituency} in the locale:

```yaml
one: "%{formatted_count} signature from %{constituency}"
other: "%{formatted_count} signatures from %{constituency}"
```

Use in the view as:
```ruby
<%= signature_count(:in_your_constituency, petition.constituency_signature_count, constituency: @constituency.name) %><br/>
```

Update `signature_count` to:

```ruby
def signature_count(key, count, options = {})
  options = options.merge({count: count, formatted_count: number_with_delimiter(count)})
  t(:"#{key}.html", scope: :"petitions.signature_counts", options)
end
```